lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -92,6 +92,14 @@ public static function get($ids) {
 
     $results = dosomething_reportback_get_reportbacks_query(['rbid' => $ids]);
 
+    if (!user_access('administer modules') && !$results) {
+      $results = dosomething_reportback_get_reportbacks_query(['rbid' => $ids], TRUE);
```

Comment:
A little confused as to why this is happening in here and passing the `TRUE` if does _not_ have user access?